flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/JdbcInputFormat.java

```diff
@@ -99,7 +99,11 @@
  * @see JdbcParameterValuesProvider
  * @see PreparedStatement
  * @see DriverManager
+ * @deprecated Please use {@link org.apache.flink.connector.jdbc.source.JdbcSource} instead. The
+ *     builder utils and parameters passing could be view {@link
```

Comment:
I am not sure what this sentence means. Is it something to do with jdbc views or relating to the viewing the link

Reply:
hi, @davidradl 

Thank you very much for your comment.
Your concerns are reasonable.
Firstly, there are two questions:  
1. There may be syntax issues with the expression. I just want to declare here that this method has expired and requires the use of a new API for development, with the new class declared here.  I'll update the sentence.

2. A comprehensive document is indeed needed here to guide developers.
As https://issues.apache.org/jira/browse/FLINK-25420 mentioned in, there are currently no plans to introduce relevant documents.

Of course, if you are willing to write usage documentation for this new API, we would greatly appreciate it.